ui/admin/app/templates/scopes/scope/storage-buckets/storage-bucket/index.hbs

```diff
@@ -16,6 +16,19 @@
     <Hds::Copy::Snippet @textToCopy={{@model.id}} @color='secondary' />
   </page.header>
 
+  <page.actions>
+    <Hds::Dropdown as |dd|>
+      <dd.ToggleButton @text={{t 'actions.manage'}} @color='secondary' />
+      {{#if (can 'delete model' @model)}}
```

Comment:
Shouldn't this be for storage bucket since we have a specific ability for the feature flag?

```suggestion
      {{#if (can 'delete storage-bucket' @model)}}
```

Reply:
yes, it should be the storage bucket model, thank you. I also moved the if condition up to wrap up the page.actions since we only have delete dropdown